### ISSUE TITLE
Threejs/Physijs Game MMO Character controlls

### ISSUE THREAD

**Issue by chilliwebs** [chandlerprall/Physijs#143]
Hey,

I'm having some issues with camera shuddering and I'm not sure how to resolve it.

Here are my theories I think the camera overall jerkyness has something to do with the physics simulation bouncing the player around slightly, but I'm not sure what to do to solve this, any help would be appreciated.

I'm working on a third person character control for a game i'm developing. I'm happy with the results so far. The character controls have a lot of neat features like: if an object is in front of the camera it will move forward so you can still see the character, however the camera stutters horribly when I rotate it to the side and then turn my player away from it. I uploaded a test on JSFiddle: http://jsfiddle.net/nA8SV/ I have only tested this in chrome, and for some reason the results part doesn't get the keypresses until you click on the white space bordering the canvas on that frame.

{also i started chrome with  --disable-web-security to ignore the cross origin}

But once you click the page the key presses work. The controls are a modified version of the orbital controls. So you can left click and rotate the view. Additionally you can use the wasd keys to move around and the camera view should return behind the player when you are moving/rotating.

I apologize for the buggyness this was very difficult to get working on JSFiddle. (But the rotation bug is happening so it at least reproduces that.)

Basically I'm trying to get my camera rotation back behind my character, so i have some code that fixes the rotation on line 250, but the camera stutters as the character moves.

Thank you!

**Comment by hellofunk** [comment]
Does Physijs provide a spring constraint? I didn't see it mentioned explicitly in your Constraints wiki. Or is "spring" physics actually created by one of the other constraint types?

**Comment by eqyiel** [comment]
I would like to know this too!
